### ISSUE TITLE
✅ 🐞 Bug Report: Loop item container not respecting min-height if HTML tag is A - [ED-11293]

### ISSUE THREAD

**Issue by tomas-eklund** [elementor/elementor#22678]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I'm using the new container-based Loop Carousel widget to create a slider. I'm making the entire slide into a big clickable area by setting the HTML tag of the outer container to A. I also set the container Min Height to 600px.

![image](https://github.com/elementor/elementor/assets/1409791/ec7f1666-8329-4cb5-a451-9c940d26bec7)

However, on the front end the entire slider collapses into this sorry mess:

![image](https://github.com/elementor/elementor/assets/1409791/a817f3b3-18d1-4427-9163-0dc4e628cfab)

After some experimenting I've concluded that I can work around the problem by adding the following custom CSS to the outer container element:

```
selector {
    display: flex;
}
```

### Steps to reproduce

1. Add a Loop Carousel
2. Create the Template
3. Add an outer container to the template
4. Set a background color/image (helps illustrate the problem) on the outer container 
5. Set a min height of the outer container to something like 600px
6. Set the outer container HTML element type to A
7. Add some content to the template
8. Save template, publish the page and view it on the front end

Demo here https://elementor.tronica.se/bug-reports/container-of-a-type/
Login can be provided.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.5.18
	PHP Version: 8.1.19
	PHP Memory Limit: 128M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 16M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.2
	Site URL: https://www.rockdale.se
	Home URL: https://www.rockdale.se
	WP Multisite: No
	Max Upload Size: 16 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: Europe/Stockholm
	Admin Email: [REDACTED]
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: sv_SE
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.13.4
		Author: Elementor.com

	Elementor Pro
		Version: 3.13.2
		Author: Elementor.com


== Features ==
	Custom Fonts: 2
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Inactive
	Improved CSS Loading: Inactive
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active by default
	Grid Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Inactive
	Editor Top Bar: Active
	Landing Pages: Inactive
	Nested Elements: Active
	Lazy Load Background Images: Inactive by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default


== Log ==
	
PHP: showing 9 of 9PHP: 2023-04-24 13:49:21 [notice X 1][/home/vhosts/rockdale.se/httpdocs/wp-content/plugins/elementor/core/upgrade/manager.php::63] version_compare(): Passing null to parameter #1 ($version1) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-24 13:50:26 [notice X 99417][/home/vhosts/rockdale.se/httpdocs/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::200] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-24 14:20:17 [warning X 1][/home/vhosts/rockdale.se/httpdocs/wp-content/plugins/elementor/core/files/file-types/svg.php::234] simplexml_load_file(): I/O warning : failed to load external entity "https://www.rockdale.se/wp-content/uploads/Notera2-Medium.svg" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-24 15:08:19 [warning X 9][/home/vhosts/rockdale.se/httpdocs/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-29 14:11:18 [warning X 17][/home/vhosts/rockdale.se/httpdocs/wp-content/plugins/elementor/modules/history/revisions-manager.php::363] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-01 13:57:51 [warning X 9][/home/vhosts/rockdale.se/httpdocs/wp-content/plugins/elementor-pro/modules/dynamic-tags/pods/tags/pods-text.php::20] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-02 10:56:05 [warning X 13][/home/vhosts/rockdale.se/httpdocs/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::171] Undefined array key "editor_post_id" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-02 14:15:16 [warning X 3][/home/vhosts/rockdale.se/httpdocs/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key "condition_type" [array (
  'trace' => '
#0: /home/vhosts/rockdale.se/httpdocs/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /home/vhosts/rockdale.se/httpdocs/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /home/vhosts/rockdale.se/httpdocs/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /home/vhosts/rockdale.se/httpdocs/wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-06-05 14:34:47 [warning X 2][/home/vhosts/rockdale.se/httpdocs/wp-content/plugins/elementor-pro/modules/query-control/classes/elementor-post-query.php::247] Undefined property: WP_Error::$taxonomy [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 20 of 202023-05-03 15:39:22 [info] elementor-pro::elementor_pro_updater Started 
2023-05-03 15:39:22 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-05-03 15:39:22 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-05-03 15:39:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.3',
  'to' => '3.12.3',
)]
2023-05-03 15:39:22 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.3',
  'to' => '3.12.3',
)]
2023-05-25 15:31:33 [info] elementor-pro::elementor_pro_updater Started 
2023-05-25 15:31:33 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-05-25 15:31:33 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-05-25 15:31:33 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.3',
  'to' => '3.13.2',
)]
2023-05-25 15:31:33 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.3',
  'to' => '3.13.2',
)]
2023-05-25 15:33:27 [info] elementor::elementor_updater Started 
2023-05-25 15:33:27 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-25 15:33:27 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-25 15:33:27 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.2',
  'to' => '3.13.3',
)]
2023-05-25 15:33:27 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.2',
  'to' => '3.13.3',
)]
2023-05-29 14:12:51 [info] elementor::elementor_updater Started 
2023-05-29 14:12:51 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-29 14:12:51 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-29 14:12:51 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.3',
  'to' => '3.13.4',
)]
2023-05-29 14:12:51 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.13.3',
  'to' => '3.13.4',
)]

JS: showing 8 of 8JS: 2023-06-02 08:54:27 [error X 1][https://www.rockdale.se/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view29620") has already been destroyed and cannot be used. 
JS: 2023-06-02 09:10:19 [error X 1][https://www.rockdale.se/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.13.4:3:1097282] Cannot read properties of null (reading 'getBoundingClientRect') 
JS: 2023-06-02 09:39:39 [error X 1][https://www.rockdale.se/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view1245") has already been destroyed and cannot be used. 
JS: 2023-06-02 12:27:55 [error X 1][https://www.rockdale.se/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view5607") has already been destroyed and cannot be used. 
JS: 2023-06-02 12:49:00 [error X 1][https://www.rockdale.se/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view120") has already been destroyed and cannot be used. 
JS: 2023-06-02 13:04:11 [error X 1][https://www.rockdale.se/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view181") has already been destroyed and cannot be used. 
JS: 2023-06-02 13:09:26 [error X 1][https://www.rockdale.se/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view12618") has already been destroyed and cannot be used. 
JS: 2023-06-02 13:32:14 [error X 2][https://www.rockdale.se/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view185") has already been destroyed and cannot be used. 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	

				

````

</details>

**Comment by nicholaszein** [comment]
Hello @tomas-eklund! 

Thank you for sharing your detailed report, we appreciate it! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by nicholaszein** [comment]
Hello @tomas-eklund! 

We are aware of this issue and our team is already working on a fix. We can’t say exactly when it'll be sorted out, but we kindly ask for your patience. Thanks for understanding! 😉

Best regards

**Comment by nicholaszein** [comment]
Hello there @tomas-eklud!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.16! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
